rnd/autogpt_server/autogpt_server/blocks/block.py

```diff
@@ -25,6 +25,8 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="45e78db5-03e9-447f-9395-308d712f5f08",
+            description="Given a code string, this block allows the verification and installation of a block code into the system.",
```

Comment:
I think this is unclear, you can provide python code?

Reply:
Yes, this is dev-only block that does RCE.
There is a description in the class. This should be disabled in the cloud version.